### PR TITLE
FFWEB-1778 add predictive basket

### PR DESCRIPTION
**Pull request by omikron-fp**
Adds "Experimental" section to left navigation. It appears only if there are pages registered for it.

Add Predictive Basket page.

### CODE REVIEW

File: markdown/3.x/en/ff-predictive-basket.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+It must be **activated** by configuring two values on **page load** as follows:
+
+```js
+document.addEventListener("ffReady", ({ factfinder }) => {
+  factfinder.communication.globalSearchParameter.version = "ng";
+  factfinder.__experimental.predictiveBasket.enable = true;
+});
+```
+
+Without this configuration the `ff-predictive-basket` element will throw errors on attempting to use it.
```
Comment (KrzysztofKozlowskiOBSS) [inline markdown/3.x/en/ff-predictive-basket.md:20]:
```suggestion
Without this configuration, the `ff-predictive-basket` element will throw errors on attempting to use it.
```

File: markdown/3.x/en/ff-predictive-basket.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+_(*) Default value is inherited from the FACT-Finder API._
+
+---
+
+Attribute | Required | Type | Default | JS property
+--------- | -------- | ---- | ------- | -----------
+blacklist | optional | String (comma-separated list) | _(empty)_ | `blacklist`
+
+With the `blacklist` attribute you can specify one or more product IDs that shall not be returned from FACT-Finder.
```
Comment (KrzysztofKozlowskiOBSS) [inline markdown/3.x/en/ff-predictive-basket.md:70]:
```suggestion
With the `blacklist` attribute, you can specify one or more product IDs that shall not be returned from FACT-Finder.
```

I'm not super sure about this one.

**Review by KrzysztofKozlowskiOBSS — APPROVED** [review]
LGTM, only some commas are missing.

**Review by KrzysztofKozlowskiOBSS — DISMISSED** [review]

File: markdown/3.x/en/ff-predictive-basket.md (hunk 1 above)
Comment (omikron-fp) [reply markdown/3.x/en/ff-predictive-basket.md]: According to some style guides starting a sentence with "With" or "Without" is not very good.
The good old "If ....." seems t be the safer bet here.

**Comment by omikron-fp** [comment]
Squashed fixup.

**Review by KrzysztofKozlowskiOBSS — APPROVED** [review]

**Pull request merged**